### PR TITLE
Issue #21 - [Mi9 - Add "lang" and/or "xml:lang" attributes to this "<html>" element]

### PR DESCRIPTION
**Pull request by Nishrith**
**What's in the PR**
* Added "lang" attribute to the html tags

### CODE REVIEW

File: .sonarcloud.properties [hunk 1]
```diff
@@ -0,0 +1 @@
+
```
Comment (munterkalmsteiner) [inline .sonarcloud.properties]: Sonarqube files should not be included in the repository.

**Review by munterkalmsteiner — CHANGES_REQUESTED** [review]
Hi,
the PR describes the change (good), but does not motivate why it is needed (bad).

Otherwise, the PR looks good, except the inclusion of the sonarqube file.

File: .sonarcloud.properties (hunk 1 above)
Comment (Nishrith) [reply .sonarcloud.properties]: removed the sonar files

**Review by munterkalmsteiner — CHANGES_REQUESTED** [review]
PR description still does not explain why the "lang" attribute is needed or good practice.

**Review by munterkalmsteiner — APPROVED** [review]
Meant to approve...

**Pull request closed without merging**